Packages/Preferences/Sources/Preferences.swift

```diff
@@ -22,6 +22,7 @@ public final class Preferences: @unchecked Sendable {
         static let authenticationLockOption = "authentication_lock_option"
         static let isDeveloperEnabled = "is_developer_enabled"
         static let isHideBalanceEnabled = "is_balance_privacy_enabled"
+        static let isAcceptedTerms = "is_accepted_terms"
```

Comment:
```suggestion
        static let isAcceptTermsEnabled = "is_accept_terms_enabled"
```

for consistency, or improve more, same below for `isAcceptedTerms`